packages/paypal-commerce-integration/src/mocks/get-paypal-sdk.mock.ts

```diff
@@ -22,6 +22,38 @@ export default function getPayPalSDKMock(): PayPalSDK {
             close: jest.fn(),
             isEligible: jest.fn(() => true),
         }),
+        CardFields: () =>
+            ({
+                isEligible: () => true,
+                CVVField: jest.fn().mockReturnValue({
+                    render: jest.fn(),
+                    clear: jest.fn(),
+                    close: jest.fn(),
+                    removeClass: jest.fn(),
+                }),
```

Comment:
You can create a variable for:
```
const cardFieldMethodsMock = {
    render: jest.fn(),
    clear: jest.fn(),
    close: jest.fn(),
     removeClass: jest.fn(),
};
```

It might help to remove duplication and reduce amount of code that we need to read every day 👨‍💻

Reply:
Done